<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string> 一梦江湖助手 v0.4</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="rewordTab">
     <attribute name="title">
      <string>悬赏</string>
     </attribute>
     <widget class="QGroupBox" name="rewordInstanceGroupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>585</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>副本</string>
      </property>
      <widget class="QCheckBox" name="szhwRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>山珍海味</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jypyRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>聚义平冤</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="jhxsRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>江湖行商</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="xygRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>羡渔港</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="wjszRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>万劫山庄</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="wrgRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>无人谷</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="myszRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>明月山庄</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="lrmcRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>落日马场</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="tybRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>铁衣堡</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="yzhjRewordCheckBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>弈中幻境</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="instanceBoxGroupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>90</y>
        <width>585</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>盒子</string>
      </property>
      <widget class="QLabel" name="szhwRewordLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>山珍海味</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="szhwRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>12</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
      </widget>
      <widget class="QLabel" name="jypyRewordLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>聚义平冤</string>
       </property>
      </widget>
      <widget class="QLabel" name="jhxsRewordLabel">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>江湖行商</string>
       </property>
      </widget>
      <widget class="QLabel" name="xygRewordLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>羡渔港</string>
       </property>
      </widget>
      <widget class="QLabel" name="wjszRewordLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>万劫山庄</string>
       </property>
      </widget>
      <widget class="QLabel" name="tybRewordLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>铁衣堡</string>
       </property>
      </widget>
      <widget class="QLabel" name="wrgRewordLabel">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>无人谷</string>
       </property>
      </widget>
      <widget class="QLabel" name="myszRewordLabel">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>明月山庄</string>
       </property>
      </widget>
      <widget class="QLabel" name="lrmcRewordLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>落日马场</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="jypyRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="jhxsRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>9</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="xygRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="wjszRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="wrgRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="myszRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="lrmcRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>30</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="tybRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>35</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>105</number>
       </property>
      </widget>
      <widget class="QLabel" name="yzhjRewordLabel">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>弈中幻境</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="yzhjRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="otherGroupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>210</y>
        <width>585</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>其他</string>
      </property>
      <widget class="QLabel" name="scanDelayRewordLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新后扫描延时</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="scanDelayRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
      <widget class="QLabel" name="popupDelayRewordLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>弹出确认框延时</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="popupDelayRewordSpinBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="rewordOutputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>411</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0;
border-radius: 5px;
background-color: #333;
color: #00ff00;
padding: 2px 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="rewordStopButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>65</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid #ff4242;
border-radius: 3px;
background-color: #fff;
color: #ff4242;
}

QPushButton:hover {
background-color: #ff4242;
color:#fff;
}</string>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rewordStartButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>330</y>
        <width>145</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-width: 0;
border-radius: 3px;
background-color: #1E90FF;
color: #FFF;
}

QPushButton:hover {
background-color: #00BFFF;
}</string>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rewordResetWindowButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>65</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;重置窗口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid #333;
border-radius: 3px;
background-color: #fff;
color: #333;
}

QPushButton:hover {
background-color: #333;
color:#fff;
}</string>
      </property>
      <property name="text">
       <string>窗口重置</string>
      </property>
     </widget>
     <widget class="QLabel" name="scanResultOutputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>341</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0;
border-radius: 5px;
background-color: #333;
color: #00ff00;
padding: 2px 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="rewordScriptStateLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0;
border-radius: 5px;
background-color: #ff4242;
color: #fff;
padding: 2px 5px;</string>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingTab">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QGroupBox" name="settingPulignGroupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>140</width>
        <height>60</height>
       </rect>
      </property>
      <property name="title">
       <string>插件</string>
      </property>
      <widget class="QPushButton" name="registerPulignButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>50</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border:1px solid #333;
border-radius: 3px;
background-color: #fff;
color: #333;
}

QPushButton:hover {
background-color: #333;
color:#fff;
}</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="uinstallPulignButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>50</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border:1px solid #ff4242;
border-radius: 3px;
background-color: #fff;
color: #ff4242;
}

QPushButton:hover {
background-color: #ff4242;
color:#fff;
}</string>
       </property>
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="settingRewordGroupBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>431</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>悬赏</string>
      </property>
      <widget class="QLabel" name="seetingRewordNumberFontPathLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>字库</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="settingRewordNumberFontPathLineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>371</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333;
border-radius: 5px;
padding-left:5px;</string>
       </property>
       <property name="text">
        <string>.\resource\reword_number.txt</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
